src/palindrome.js

```diff
@@ -0,0 +1,13 @@
+export default function isPalindrome(str){
+  let reversedStr = str
+  reversedStr = reversedStr.replace(/[^a-zA-z]/g, '')
+  reversedStr = reversedStr.toLowerCase().split('').reverse()
+  str = str.replace(/[^a-zA-Z]/g, '')
+  str = str.toLowerCase().split('')
+  for (var i = 0; i < str.length; i++) {
+    if(str[i] !== reversedStr[i]){
+      return false
+    }
+  }
+  return true
+}
```

Comment:
Good job.

You could also avoid the for loop and make it a little more concise and readable by `join`ing the reversed string array back into a string and then comparing the two strings directly like this:

```
export default function isPalindrome(str){
  let reversedStr = str
  reversedStr = reversedStr.replace(/[^a-zA-Z]/g, '')
  reversedStr = reversedStr.toLowerCase().split('').reverse().join('')
  str = str.toLowerCase()

  return reversedStr === str
}
```